### DISCUSSION TITLE
Welcome to Cohort-Backend-Dev Discussions!

### DISCUSSION THREAD

**Discussion by collinsoden** [Cohort6-lab/Cohort-Backend-Dev#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  📢 **Announcement:** Discussions is available! Here, you can ask questions, make your suggestions and find help from community members.

**Comment by obed4life** [comment]
Good day

**Reply by KebokwuFavour** [reply]
Same here

**Comment by akinolaemmanuel49** [comment]
What is this report about? What is the stack PHP or python (Django)? What is it supposed to do?

**Comment by obed4life** [comment]
Please ask prol I don't really know but I guess php frameworks Laravel and
python are the two framework that we are using for the backend
